pkg/agent/testdata/RawUbuntuContainerd/line23.sh

```diff
@@ -105,7 +105,7 @@ installContainerRuntime() {
 
 installMoby() {
     CURRENT_VERSION=$(dockerd --version | grep "Docker version" | cut -d "," -f 1 | cut -d " " -f 3 | cut -d "+" -f 1)
-    local MOBY_VERSION="19.03.12"
+    local MOBY_VERSION="19.03.14"
```

Comment:
why is this the only test data change?

Reply:
discussed offline, looks like containerd testdata for this case isn't updating. will leave it as-is and let Erik take a look later. naive attempt to fix it didn't seem to produce correct testdata